serpentTools/parsers/sensitivity.py

```diff
@@ -331,6 +331,10 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
         mat: None or str or list of strings
             Plot sensitivities due to these materials. Passing ``None``
             will plot against all materials.
+        mevscale: string
+            Flag for plotting energy grid in MeV units. Default is ``False``.
+        egrid : numpy.array
```

Comment:
We are working on improving the docs to fit the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html) more rigorously. To this end, I would make the change
```suggestion
        egrid : numpy.array, optional
```